Makefile

```diff
@@ -51,6 +51,49 @@ dev: ## dev
 		$(eval var_file=$(env))
 		$(eval backend_config=-backend-config="key=$(env)/app.tfstate")
 
+
+.PHONY: development_aks
+development_aks:
+	$(eval include global_config/development.sh)
+
+
+.PHONY: set-key-vault-names
+set-key-vault-names:
```

Comment:
We won't use key vault so this can be removed

Reply:
resolved